apps/ledger-live-mobile/src/families/mina/ScreenEditMemo.tsx

```diff
@@ -0,0 +1,119 @@
+import invariant from "invariant";
+import React, { useCallback, useState } from "react";
+import { View, StyleSheet, ScrollView } from "react-native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { useSelector } from "react-redux";
+import { useTranslation } from "react-i18next";
+import i18next from "i18next";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { useIsFocused, useTheme } from "@react-navigation/native";
+import KeyboardView from "~/components/KeyboardView";
+import Button from "~/components/Button";
+import { ScreenName } from "~/const";
+import { accountScreenSelector } from "~/reducers/accounts";
+import TextInput from "~/components/FocusedTextInput";
+import { BaseComposite, StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
+import { SendFundsNavigatorStackParamList } from "~/components/RootNavigator/types/SendFundsNavigator";
+import { SignTransactionNavigatorParamList } from "~/components/RootNavigator/types/SignTransactionNavigator";
+import { SwapNavigatorParamList } from "~/components/RootNavigator/types/SwapNavigator";
+
+type NavigationProps = BaseComposite<
+  StackNavigatorProps<
+    SendFundsNavigatorStackParamList | SignTransactionNavigatorParamList | SwapNavigatorParamList,
+    ScreenName.MinaEditMemo
+  >
+>;
+
+function MinaEditMemo({ navigation, route }: NavigationProps) {
+  const isFocused = useIsFocused();
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+  const { account } = useSelector(accountScreenSelector(route));
+  invariant(account, "account is required");
+  const [memo, setMemo] = useState(route.params?.transaction.memo);
+  const onChangeMemoValue = useCallback((str: string) => {
+    setMemo(str === "" ? undefined : str);
+  }, []);
+  const onValidateText = useCallback(() => {
+    const bridge = getAccountBridge(account);
+    const { transaction } = route.params;
+    // @ts-expect-error FIXME: No current / next navigation params?
```

Comment:
?

Reply:
I remember this from one of the first integrations where we copied a few things, I can try to fix it for mina but I see this FIXME in multiple coins 😓